backend/src/test/java/com/bakdata/conquery/integration/json/AbstractQueryEngineTest.java

```diff
@@ -64,7 +64,8 @@ public void executeTest(StandaloneSupport standaloneSupport) throws IOException,
 		}
 
 		//check result info size
-		ResultInfoCollector resultInfos = managed.collectResultInfos();
+		ResultInfoCollector resultInfos = managed.collectResultInfos(new PrintSettings(false,Locale.ROOT, namespaces));
```

Comment:
Die settings kommen zwar weg, aber in den Tests habe ich im normal fall Locale.ENGLISH genommen